feature-libs/product/configurators/rulebased/components/show-more/configurator-show-more.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="cx-show-more" *ngIf="text">
+  <div #textContent class="show-more-text">
+    {{ textToShow }}
+    <span class="show-more-button" *ngIf="showMore$ | async">
```

Comment:
we shouldn't use block elements inside of inline elements (`button` inside of `span`).

Reply:
changed